src/GovUk.Education.ExploreEducationStatistics.Content.Model/ReleaseType.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+{
+    public class ReleaseType
+    {
+        public Guid Id { get; set; }
+
+        [Required] public string Title { get; set; }
+
+        protected bool Equals(ReleaseType other)
+        {
+            return Id.Equals(other.Id);
+        }
+
+        public override bool Equals(object obj)
+        {
+            if (ReferenceEquals(null, obj)) return false;
+            if (ReferenceEquals(this, obj)) return true;
+            if (obj.GetType() != this.GetType()) return false;
+            return Equals((ReleaseType) obj);
+        }
+
+        public override int GetHashCode()
+        {
+            return Id.GetHashCode();
+        }
```

Comment:
Shouldn't necessary to add equals methods in C#

Reply:
Other entities I've seen have done it based on the id - I assume so that during editing we know that the items are still the same thing? I'm certainly happy enough to remove it if that is not the model we're going for - and actually it would be my preference I think

Reply:
Discussed - I will remove the equality overrides

Reply:
Done